V2 Optimization Issue when Webpack Mode Production

**Environment**
✔ Getting NativeScript components versions information...
✔ Component nativescript has 6.2.2 version and is up to date.
✔ Component tns-core-modules has 6.2.3 version and is up to date.
✔ Component tns-android has 6.2.0 version and is up to date.
✔ Component tns-ios has 6.2.0 version and is up to date.

**Describe the bug**
When building a release version of the application for iOS, I noticed that I was getting this message:
`CONSOLE ERROR file:///node_modules/@nativescript/core/trace/trace.js:174:30: Error: Css styling failed: Error: undefined:11:16959: property missing ':'`.  There are also several visual css problems with the app.

Upon further investigation, I isolated the issue as a result of some sort of optimization that webpack is doing in 'production' mode as it relates to this file:
`https://github.com/NativeScript/theme/blob/master/src/scss/core/colors/_index.scss`

Specifically, it has to do with the `transparent: transparent` setting within the `$color` variable.  Whatever webpack optimization doesn't like that.  I noticed that changing it to `transparent: rgba(0, 0, 0, 0)` causes everything to work properly.

I tried to do as much due diligence as I could to isolate the exact issue and cause.  Please let me know if there is any more information I can provide or anything I can do to help.  Thanks!

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,12 @@
-<a name="2.3.0"></a>
-## [2.3.0](https://github.com/NativeScript/theme/compare/v2.2.1...v2.3.0) (2020-01-13)
+<a name="2.3.2"></a>
+## [2.3.2](https://github.com/NativeScript/theme/compare/v2.2.1...v2.3.2) (2020-02-20)
+
+### Fixes
+
+* CSS variables output with **node-sass** is causing crashes
+
+<a name="2.3.1"></a>
+## [2.3.1](https://github.com/NativeScript/theme/compare/v2.2.1...v2.3.1) (2020-01-13)
 
 ### Features
 

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@nativescript/theme",
-  "version": "2.3.1",
+  "version": "2.3.2",
   "description": "Telerik NativeScript Core Theme",
   "author": "Telerik <support@telerik.com>",
   "homepage": "https://www.nativescript.org",
@@ -14,14 +14,14 @@
   "nativescript": {
     "id": "org.nativescript.theme",
     "tns-ios": {
-      "version": "6.3.0"
+      "version": "6.4.0"
     },
     "tns-android": {
-      "version": "6.3.1"
+      "version": "6.4.1"
     }
   },
   "dependencies": {
-    "@nativescript/core": "~6.3.2",
+    "@nativescript/core": "~6.4.1",
     "@nativescript/theme": "./src",
     "bootstrap": "4.3.1",
     "nativescript-picker": "~2.1.2",
@@ -31,7 +31,7 @@
     "nativescript-ui-sidedrawer": "~8.0.0",
     "nativescript-ui-dataform": "~6.0.0",
     "nativescript-ui-listview": "~8.0.1",
-    "tns-core-modules": "~6.3.2"
+    "tns-core-modules": "~6.4.1"
   },
   "devDependencies": {
     "@babel/core": "7.7.7",
@@ -44,9 +44,9 @@
     "lazy": "1.0.11",
     "nativescript-dev-webpack": "1.4.1",
     "resolve-url-loader": "3.1.0",
-    "sass": "1.24.3",
+    "sass": "1.25.0",
     "sass-lint": "1.13.1",
-    "sass-loader": "8.0.0",
+    "sass-loader": "8.0.2",
     "speed-measure-webpack-plugin": "1.3.1",
     "webpack": "4.40.2",
     "webpack-bundle-analyzer": "3.5.0",

--- a/src/package.json
+++ b/src/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@nativescript/theme",
-  "version": "2.2.1",
+  "version": "2.3.2",
   "description": "Telerik NativeScript Core Theme",
   "author": "Telerik <support@telerik.com>",
   "main": "index",

--- a/src/scss/core/colors/_index.scss
+++ b/src/scss/core/colors/_index.scss
@@ -19,12 +19,12 @@ $border-color: $focus !default;
 
 $colors:  (
     // Core colors
-    black: $black,
-    white: $white,
+    "black": $black,
+    "white": $white,
     grey: $grey,
     grey-light: $grey-background,
     charcoal: $charcoal,
-    transparent: transparent,
+    "transparent": transparent,
 
     // Accents
     aqua: #00caab,

